### PR TITLE
Rename Chronicle to Google SecOps

### PR DESCRIPTION
**Pull request by rarsan**

### CODE REVIEW

File: CONTRIBUTING.md
Comment (railroadmanuk) [inline CONTRIBUTING.md:1]: Image linked in line 53 need updating

File: README.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ The security use cases below are grouped in 6 categories depending on underlying
...
+**Caution:** CSA is not meant to be a comprehensive set of threat detections, but a collection of community-contributed samples to get you started with detective controls. Use CSA in your threat detection and response capabilities (e.g. [Security Command Center](https://cloud.google.com/security-command-center), Google Security Operations, BigQuery, Siemplify, or third-party SIEM) in conjunction with threat prevention capabilities (e.g. [Security Command Center](https://cloud.google.com/security-command-center), [Cloud Armor](https://cloud.google.com/armor), [BeyondCorp](https://cloud.google.com/beyondcorp)). To learn more about Google’s approach to modern Security Operations, check out the [Autonomic Security Operations whitepaper](https://services.google.com/fh/files/misc/googlecloud_autonomicsecurityoperations_soc10x.pdf).
```
Comment (railroadmanuk) [inline README.md:26]: Can we remove Siemplify from this list as well please

File: CONTRIBUTING.md
Comment (rarsan) [reply CONTRIBUTING.md:1]: Image replaced, and contributing guide updated.

File: README.md (hunk 1 above)
Comment (rarsan) [reply README.md]: Good catch. Updated.

**Comment by rarsan** [comment]
Rename Chronicle to Google SecOps, update README and Contributing guide

**Pull request merged**